CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Thunderdome Changelog
 
 ## Unreleased Changes
+* Added `Index::slot` for extracting the slot portion of an index.
+* Added `Arena::contains_slot` for checking whether a slot is occupied in a given `Arena` and resolving its `Index` if so.
+* Added `Arena::get_by_slot` and `Arena::get_by_slot_mut` for retrieving an entry by its slot, ignoring generation.
+* Renamed `Arena::remove_entry_by_slot` to `Arena::remove_by_slot`, making it public.
```

Comment:
From the perspective of the public API, this was just an addition! I suspect that most users of Thunderdome besides us didn't know about `remove_entry_by_slot`. 😏

```suggestion
* Added `Arena::remove_by_slot` for removing an entry by its slot, ignoring generation.
```